 .../src/buildcraft/transport/RenderPipe.java       |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/buildcraft/buildcraft_client/src/net/minecraft/src/buildcraft/transport/RenderPipe.java b/buildcraft/buildcraft_client/src/net/minecraft/src/buildcraft/transport/RenderPipe.java
index 67e713b..89017a7 100755
--- a/buildcraft/buildcraft_client/src/net/minecraft/src/buildcraft/transport/RenderPipe.java
+++ b/buildcraft/buildcraft_client/src/net/minecraft/src/buildcraft/transport/RenderPipe.java
@@ -464,7 +464,7 @@ public class RenderPipe extends TileEntitySpecialRenderer {
                     GL11.glTranslatef(f6, f9, f11);
                 }
                 float f7 = 1.0F;
-                renderBlocks.renderBlockAsItem(Block.blocksList[itemstack.itemID], itemstack.getItemDamage(), f7);
+                renderBlocks.renderBlockOnInventory(Block.blocksList[itemstack.itemID], itemstack.getItemDamage(), f7);
                 GL11.glPopMatrix();
             }
 
